XML/Other/ReplaceableTestParameters.xml

```diff
@@ -677,4 +677,8 @@ Scenario 2 : You need to run all the performance tests quickly.
 		<ReplaceThis>MAX_CPU_OFFLINEONLINE</ReplaceThis>
 		<ReplaceWith>10</ReplaceWith>
 	</Parameter>
+	<Parameter>
+		<ReplaceThis>XDPDump_Repo_URL</ReplaceThis>
```

Comment:
To make code consistent, use upper letters for the whole string.

Reply:
Fixed.